product/toolbars/containers_center_tb.yaml

```diff
@@ -52,6 +52,12 @@
     :title: Monitoring
     :text: Monitoring
     :items:
+    - :button: container_timeline
+      :image: timeline
+      :text: "Timelines"
+      :title: "Show Timelines for this Container"
+      :url: '/explorer'

```

Comment:
> @dkorn this warning message does not seem to be related to timelines at all.

I know, but there was no other ERROR/WARN in development.log or evm.log

> did you restart server after removing :url: '/explorer',

I did 

> i am curious how is it going thru x_button method if you are sending up url as explorer. 

I'm not sure it's going through this method?